main.js

```diff
@@ -16,51 +16,55 @@ function generateId() {
 }
 
 function storeData() {
-  localStorage.setItem('books', JSON.stringify(bookList));
+  localStorage.setItem('bookList', JSON.stringify(bookList));
 }
 
 const add = document.getElementById('add');
 
-function Book(title, author, id) {
-  this.title = title;
-  this.author = author;
-  this.id = id;
-}
+class Books {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 
-function createBook({ title, author, id }) {
-  const newBook = document.createElement('div');
-  const titlePara = document.createElement('p');
-  const authorPara = document.createElement('p');
-  const remove = document.createElement('button');
-  remove.setAttribute('id', id);
-  const line = document.createElement('hr');
-  titlePara.textContent = title;
-  authorPara.textContent = author;
-  remove.textContent = 'Remove';
+  createBook({ title, author, id }) {
+    const newBook = document.createElement('div');
+    const titlePara = document.createElement('p');
+    const authorPara = document.createElement('p');
+    const remove = document.createElement('button');
+    remove.setAttribute('id', this.id);
+    titlePara.textContent = this.title;
+    authorPara.textContent = this.author;
+    const completeBook = `"${title}" by ${author}`;
+    remove.textContent = 'Remove';
+    newBook.classList.toggle('grey', id % 2 !== 0);
+    newBook.classList.toggle('white', id % 2 === 0);
+    newBook.classList.add('position');
+    remove.classList.add('remove');
 
-  newBook.append(titlePara);
-  newBook.append(authorPara);
-  newBook.append(remove);
-  newBook.append(line);
+    newBook.append(completeBook);
+    newBook.append(remove);
 
-  remove.addEventListener('click', (e) => {
-    e.preventDefault();
-    newBook.remove();
-    bookList = bookList.filter((book) => book.id !== id);
-    storeData();
-  });
-  return newBook;
+    remove.addEventListener('click', (e) => {
+      e.preventDefault();
+      newBook.remove();
+      bookList = bookList.filter((book) => book.id !== id);
+      storeData();
+    });
+    return newBook;
+  }
 }
 
 function renderBooks() {
-  const bookSection = document.getElementById('bookSection');
+  const bookSection = document.querySelector('.box');
   const bookListElement = document.createElement('div');
   bookListElement.id = 'bookList';
   bookList.forEach((book) => {
-    const bookElem = createBook(book);
-    bookListElement.appendChild(bookElem);
+    const bookElem = new Books(book.title, book.author, book.id);
+    const bookElement = bookElem.createBook(bookElem);
+    bookSection.append(bookElement);
   });
-  bookSection.appendChild(bookListElement);
 }
```

Comment:
- [ ] please as the requirements you should also refactor this function into class so it should a method's class I encourage you to do that